index.html

```diff
@@ -324,38 +321,33 @@ <h2 class="abtMe_Header">About me </h2>
 
            <!--About Section-->
 
-    <div class="contact_form" id="contact_Me">
-       <div class="form_header">
-        <h1 id="contact_header">I'm always interested in hearing about new projects, so if you'd like to chat 
-            please get in touch.</h1></div>
-            <div class="form_container">
-            <form action="https://formspree.io/f/xqkjqrwa" method="post" id="formSubmit">
-                             <label for="user_name" name="full_name"></label>
-             <input type="text" maxlength="30" id="user_name" placeholder="Full name" required>
-                      <img src="./assets/images/underInput.png" id="under_input">
-                        <label for="user_email" name="Useremail"></label>
-                      
-                          <input type="email" id="email"placeholder="Email address" required>
-                       
-                            <img src="./assets/images/underInput.png"  id="under_input">
+          <div class="contact_form" id="contact_Me">
+            <div class="form_header">
+             <h1 id="contact_header">I'm always interested in hearing about new projects, so if you'd like to chat 
+                 please get in touch.</h1></div>
+                 <div class="form_container">
+                 <form action="https://formspree.io/f/xqkjqrwa" method="post" id="formSubmit">
+                                  <label for="name" name="full_name"></label>
+                  <input type="text" maxlength="30" id="name" placeholder="Full name" required>
+                           <img src="./assets/images/underInput.png" id="under_input">
+                             <label for="email" name="Useremail"></label>
                            
-                                <label for="user_comment" name="comment"></label> 
-                         <textarea rows="7" 
-                            cols="50" 
-                            id="user_comment" 
-                            maxlength="500" 
-                            name="comment" 
-                            placeholder="Enter text here" 
-                            required>
-                           </textarea>
-                       <label for="button_label"></label>
-                       <div id="showError">
+                               <input type="email" id="email"placeholder="Email address" required>
+                            
+                                 <img src="./assets/images/underInput.png"  id="under_input">
+                                 <div id="showError">
+     
+                                 </div>
+                                                           <label for="comment" name="comment"></label> 
+     
+      <textarea rows="7" cols="50" id="comment" maxlength="500" name="comment" placeholder="Enter text here" required>
+     </textarea>
```

Comment:
- [ ] For some reason, I am able to data to local storage without filling out the comment which must also pass validation. Kindly look into it and fix.

![validation-error](https://user-images.githubusercontent.com/26597432/197153561-37298d83-03a4-4e1f-9329-22360857969c.png)

Reply:
Thank you bro for your feedback.It is not passing empty comment on my machine.Do you mind if you guide me to what you want ?